docs/content/v2025.1/best-practices-operations/ysql-yb-enable-cbo.md

```diff
@@ -0,0 +1,76 @@
+
+The CBO is controlled using the [yb_enable_cbo](../../reference/configuration/yb-tserver/#yb-enable-cbo) configuration parameter. The `yb_enable_cbo` parameter provides a heuristic-based optimizer mode that operates independently of table analysis. This allows you to continue using the system without unexpected plan changes during the transition to cost-based optimization, and to selectively enable this mode for specific connections if needed.
+
+Note that `yb_enable_cbo` replaces the `yb_enable_optimizer_statistics` and `yb_enable_base_scans_cost_model` parameters, which will be deprecated and removed in a future release.
+
+## Options
+
+The following table shows `yb_enable_cbo` settings and the equivalent using the replaced parameters.
+
+| yb_enable_cbo | Old Settings | Use of Statistics |
+| :--- | :--- | :--- |
+| on | yb_enable_statistics: N/A<br>yb_enable_base_scans_cost_model: on | Yes |
+| off | yb_enable_statistics: off<br>yb_enable_base_scans_cost_model: off | No (new behavior) |
+| legacy_mode | yb_enable_statistics: off<br>yb_enable_base_scans_cost_model: off | Partial<sup>*</sup> |
+| legacy_stats_mode | yb_enable_statistics: on<br> yb_enable_base_scans_cost_model: off | Yes |
+
+<sup>*</sup> `legacy_mode` is the default. `legacy_mode` uses only reltuples for index scan nodes, and both reltuples and the column statistics for other operations such as joins, GROUP BY, and more.
+
+## Recommended settings
+
+| Scenario | Tables analyzed | Setting |
+| :--- | :--- | :--- |
+| New installation, migrating from another system | N/A | on |
+| Using CBO. yb_enable_base_scans_cost_model = on | Yes | on |
+| Using default settings (yb_enable_optimizer_statistics = off, yb_enable_base_scans_cost_model = off) | No | off |
+| Using default settings (yb_enable_optimizer_statistics = off, yb_enable_base_scans_cost_model = off) | Yes | legacy_mode |
+| yb_enable_optimizer_statistics = on,  yb_enable_base_scans_cost_model = off | Yes | legacy_stats_mode |
+
+## Turning on CBO
+
+When enabling the cost models, ensure that packed row for colocated tables is enabled by setting the `--ysql_enable_packed_row_for_colocated_table` flag to true.
+
+### If the tables are not analyzed
+
+1. Set `yb_enable_cbo` to `off`.
+
+    This setting ensures that the heuristic-based optimizer mode does not use the statistics inconsistently while tables are being analyzed. It's important to ensure that each client connection reflects the updated setting after the parameter change.
+
+    For example, restart the servers with the parameter specified via the `ysql_pg_conf_csv` TServer flag, or use the `ysql_yb_enable_cbo` flag.
```

Comment:
Can you link to the flags - https://deploy-preview-27721--infallible-bardeen-164bc9.netlify.app/v2025.1/reference/configuration/yb-tserver/#cost-based-optimizer-flag